plugin/api.go

```diff
@@ -107,6 +107,9 @@ type API interface {
 	// The status parameter can be: "online", "away", "dnd", or "offline".
 	UpdateUserStatus(userId, status string) (*model.Status, *model.AppError)
 
+	// UpdateUserActive updates status of a user whether active or not.
```

Comment:
Can you please add information about the minimum server version here? You can take a look at the other methods for reference. It should be like:
```go
//
// Minimum server version: 5.6
```

@cpanato Can this PR go into 5.6.0?

Reply:
Updated it

Reply:
Also, suggest:
```suggestion
	// UpdateUserActive deactivates or reactivates a user.
```

Reply:
@lieut-data Changed the comments